### PR TITLE
Azure terraform

### PR DESCRIPTION
**Pull request by ChrisSchreiber**
- Add Azure Terraform image
- Publish image to GHCR instead of ECR

### CODE REVIEW

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -9,19 +9,13 @@ jobs:
...
-        role-duration-seconds: 1200
-    - id: ecr
-      uses: liatrio/github-actions/ecr@master
     - id: skaffold
       uses: liatrio/github-actions/skaffold-build@master
       with:
-        default_repo: ${{ steps.ecr.outputs.registry }}
+        default_repo: ghcr.io/liatrio
+        docker_registry: ghcr.io/liatrio
+        docker_username: chrisjblackburn
```
Comment (mrparkers) [inline .github/workflows/push.yml:17]: shouldn't we be able to use the github actions pipeline user here?  seems odd to publish it using someone's personal access token

Comment (ChrisSchreiber) [reply .github/workflows/push.yml]: It needs to be the repository owner but I forgot there is a variable for that. I changed it to use that.

**Review by mrparkers — APPROVED** [review]

File: builder-image-aws/Dockerfile [hunk 2]
```diff
@@ -1,5 +1,7 @@
 FROM alpine:3.9
 
+LABEL org.opencontainers.image.source https://github.com/liatrio/builder-images
```
Comment (RobertKelly) [inline builder-image-aws/Dockerfile:3]: Was looking at alternatively using the label flag in the skaffold build but that's probably only for kaniko builds.  No problem, this option works well.

File: builder-image-azure-terraform/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+RUN wget -O /tmp/go.tar.gz https://golang.org/dl/go${GO_VERSION}.linux-amd64.tar.gz && \
+    tar -C /usr/local -xzf /tmp/go.tar.gz && \
+    rm /tmp/go.tar.gz && \
+    go version
+
+# Install tfenv and setup gpg verification
+RUN mkdir /opt/tfenv && \
+    git clone https://github.com/tfutils/tfenv.git /opt/tfenv/.tfenv && \
+    chmod -R ago+w /opt/tfenv/ && \ 
+    echo 'PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/opt/tfenv/.tfenv/bin"' > /etc/environment && \
```
Comment (RobertKelly) [inline builder-image-azure-terraform/Dockerfile:51]: games?  `/usr/games`. ;)

File: builder-image-azure-terraform/README.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Builder Image Azure Terraform
+
+Image for applying a Terraform plan to an Azure environment.
+
+## Tools
+
+- Azure CLI
+- GO: 1.15.8
```
Comment (RobertKelly) [inline builder-image-azure-terraform/README.md:8]: May I suggest go 1.16?

Comment (ChrisSchreiber) [reply builder-image-azure-terraform/README.md]: 👍

**Pull request merged**